Resources/Prototypes/Entities/Structures/Machines/fax_machine.yml

```diff
@@ -0,0 +1,80 @@
+﻿- type: entity
+  parent: BaseMachinePowered
+  id: FaxMachineBase
+  name: long range fax machine
+  description: Bluespace technologies on the application of bureaucracy.
+  components:
+  - type: Sprite
+    sprite: Structures/Machines/fax_machine.rsi
+    netsync: false
+    layers:
+    - state: icon
+      map: ["base"]
+  - type: Icon
+    sprite: Structures/Machines/fax_machine.rsi
+    state: icon
+  - type: Appearance
+  - type: Physics
+    bodyType: Static
+  - type: Fixtures
+    fixtures:
+    - shape:
+        !type:PhysShapeAabb
+        bounds: "-0.25,-0.4,0.25,0.4"
+      density: 25
+      mask:
+      - MachineMask
+      layer:
+      - MachineLayer
+  - type: ActivatableUI
+    key: enum.FaxUiKey.Key
+  - type: ActivatableUIRequiresPower
+  - type: UserInterface
+    interfaces:
+    - key: enum.FaxUiKey.Key
+      type: FaxBoundUi
+  - type: ApcPowerReceiver
+    powerLoad: 250
+  - type: FaxMachine
+    paperSlot:
+      insertSound: /Audio/Machines/scanning.ogg
+      ejectSound: /Audio/Machines/tray_eject.ogg
+      whitelist:
+        components:
+          - Paper
+  - type: GenericVisualizer
+    visuals:
+      enum.PowerDeviceVisuals.Powered:
+        base:
+          True: { state: idle }
+          False: { state: icon }
+      enum.FaxMachineVisuals.VisualState:
+        base:
+          Inserting: { state: inserting }
+          Printing: { state: printing }
+  - type: ItemSlots
+  - type: DeviceNetwork
+    deviceNetId: Wireless
+    receiveFrequencyId: FaxCentcom
+    transmitFrequencyId: FaxCentcom
+
+# Special
+- type: entity
+  parent: FaxMachineBase
+  id: FaxMachineCentcom
+  name: centcom long range fax machine
+  suffix: Centcom
+  components:
+  - type: FaxMachine
+    notifyAdmins: true
+
+- type: entity
+  parent: FaxMachineBase
+  id: FaxMachineSyndie
+  name: syndicate long range fax machine
+  suffix: Syndicate
+  components:
+  - type: FaxMachine
+    responsePings: false
+    emagged: true
+    notifyAdmins: true
```

Comment:
This is just an idea but you could color these differently or smth

Reply:
![image](https://user-images.githubusercontent.com/14136326/202339395-032e91e6-f75f-4aa9-8cd3-de219c3a5a95.png)

Reply:
Nice 👍